.mailmap

```diff
@@ -0,0 +1,37 @@
+Julien Tierny <julien.tierny@gmail.com> Julien J Tierny <julien.tierny@gmail.com>
+Julien Tierny <julien.tierny@gmail.com> Julien Tierny <julien@richard.nation>
+Julien Tierny <julien.tierny@gmail.com> Julien J Tierny <julien@vger.lip6.fr>
+Julien Tierny <julien.tierny@gmail.com> Julien Tierny <julien.tiernygmail.com>
+Julien Tierny <julien.tierny@gmail.com> julien.tierny@sorbonne-universite.fr <julien.tierny@gmail.com>
+
+Pierre Guillou <pierre.guillou@lip6.fr> Pierre Guillou <guillou@cri.ensmp.fr>
+Pierre Guillou <pierre.guillou@lip6.fr> pierre-guillou <47317188+pierre-guillou@users.noreply.github.com>
+
```

Comment:
Would it be possible to add:


Charles Gueunet <charles.gueunet@gmail.com> Charles Gueunet <charles.gueunet@lip6.fr>
Charles Gueunet <charles.gueunet@gmail.com> Charles Gueunet <charles.gueunet@kitware.com>
Charles Gueunet <charles.gueunet@gmail.com> Gueunet Charles <charles.gueunet@gmail.com>
Charles Gueunet <charles.gueunet@gmail.com> charles gueunet <charles.gueunet@gmail.com>

(wherever you seems it fit)

Reply:
No problem!